Adds more clarity around the current user object on ftl theme pages (#1492)
FusionAuth/fusionauth-site: site/docs/v1/tech/themes/template-variables.adoc

The OAuth v2.0 `client_id` parameter. This is synonymous with FusionAuth's Application Id.

[field]#currentUser# [type]#[User]# [since]#Available since 1.30.0#::

> When there is an active SSO session, this variable will contain the currently logged in user. When an SSO session does not yet exist, this variable will be undefined.  In other words, this variable availability is synonymous with the `Keep me signed in` login option.

Reviewer: ```suggestion
When there is an active SSO session, this variable will contain the currently logged in user. When an SSO session does not yet exist, this variable will be null. If the user has not checked the `Keep me signed in` option, there is no SSO session and this variable will be null.
```

> When there is an active SSO session, this variable will contain the currently logged in user. When an SSO session does not yet exist, this variable will be null. If the user has not checked the `Keep me signed in` option, there is no SSO session and this variable will be null.

Reviewer 2: null should be `null`.

Author: Updated and merged in a separate PR. Thanks!
